addons/website_event_migration_layer/static/src/js/website_event_track.js

```diff
@@ -0,0 +1,62 @@
+/** @odoo-module **/
+
+import publicWidget from "@web/legacy/js/public/public_widget";
+
+publicWidget.registry.websiteEventTrackML = publicWidget.Widget.extend({
+    selector: '.o_wevent_event',
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    start: function () {
+        this._super.apply(this, arguments).then(() => {
+            this.agendas = this.target.getElementsByClassName('o_we_online_agenda');
+            if (this.agendas.length > 0) {
+                this._computeAgendasSize();
+
+                Array.from(this.agendas).forEach((agenda) => {
+                    agenda.addEventListener("scroll", (event) => {
+                        this._onAgendaScroll(agenda, event);
+                    });
+                });
+
+                window.addEventListener("resize", () => {
+                    this._computeAgendasSize();
+                });
+            }
+        })
+    },
+
+    /**
+     * @private
+     */
+    _computeAgendasSize: function () {
```

Comment:
would rather call it "updateScrollOverlay" or something. It doesn't really compute any size.